app/BUILD.bazel

```diff
@@ -407,6 +407,12 @@ in order to keep Gradle building.
      """,
 ) for file in BINDING_ADAPTERS_WITH_RESOURCE_IMPORTS]
 
+CONFETTI = [
```

Comment:
From my understanding these presenter & assembler files are lumped in with the high-level app modules and so it seemed incorrect to be adding a confetti dependency for such a higher module when it was only these files that used it. That said, I agree this isn't the right idea and need to reason a bit more about how to add the dependency.

Reply:
Oh, I actually think we should be keeping the confetti view as part of the top-level app module. Let's not try splitting it out now since teasing apart these dependencies can be challenging, and may require substantial additional work that's unrelated to this PR (we actually have a separate whole project around splitting up these libraries).

Reply:
Done

I added confetti in the "annotations" library but please let me know if this isn't the best place for it. I removed the individual confetti library

Reply:
I don't think that's the right place--created another comment thread to discuss.